src/client/clientmap.cpp

```diff
@@ -840,9 +840,7 @@ void ClientMap::updateDrawListShadow(v3f shadow_light_pos, v3f shadow_light_dir,
 	v3s16 p_blocks_max;
 	getBlocksInViewRange(cam_pos_nodes, &p_blocks_min, &p_blocks_max, radius + length);
 
-	std::vector<v2s16> blocks_in_range;
-
-	for (auto &i : m_drawlist_shadow) {
+	for (const auto &i : m_drawlist_shadow) {
```

Comment:
Useless reference to iterator.

Reply:
@SmallJoker, fix it https://github.com/minetest/minetest/pull/12704/commits/7b18f1da0cf9bc70b334d51fc03ca93000e70811